Sources/Brave/WebFilters/AdblockRustEngine.swift

```diff
@@ -30,7 +30,6 @@ extension AdblockEngine {
     }
     
     guard let requestHost = requestURL.host, let sourceHost = sourceURL.host else {
-      assertionFailure("You must provide absolute paths for `requestURL` and `sourceURL`")
```

Comment:
As discussed in Slack https://bravesoftware.slack.com/archives/GB9CJGQV6/p1677704768849219